examples/autoscaling/effective-hpa-cron-local.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: autoscaling.crane.io/v1alpha1
+kind: EffectiveHorizontalPodAutoscaler
+metadata:
+  name: php-apache-local
+spec:
```

Comment:
How to ensure the timezone in metric-adapter container is sync to cluster and node? should we mount timezone config to metric-adapter ?

Reply:
There are three ways to setting the container timezone.  it is depends on how to deploy the service. 
1. setting the timezone when build the docker image.
2. use host path to pass the host's timezone to container.
3. use env to set the pod yaml to set the timezone.

currently i use build image default timezone. the first one.  I think use 3 is better.